flocker/control/httpapi.py

```diff
@@ -414,6 +420,70 @@ def state_datasets(self):
             del dataset[u"deleted"]
         return datasets
 
+    @app.route("/configuration/containers", methods=['POST'])
+    @user_documentation(
+        """
+        Create and start a new container.
+        """,
+        examples=[
+            u"create container",
+            u"create container with duplicate name",
+        ]
+    )
+    @structured(
+        inputSchema={
+            '$ref': '/v1/endpoints.json#/definitions/configuration_container'},
+        outputSchema={
+            '$ref': '/v1/endpoints.json#/definitions/configuration_container'},
+        schema_store=SCHEMAS
+    )
+    def create_container_configuration(self, host, name, image):
+        """
+        Create a new dataset in the cluster configuration.
+
+        :param unicode host: The address of the node on which the container
+            will run.
+
+        :param unicode name: A unique identifier for the container.
```

Comment:
Indicate that it is unique within the Flocker cluster, not just on the named host.